assets/global.js

```diff
@@ -417,6 +417,11 @@ customElements.define('menu-drawer', MenuDrawer);
 class HeaderDrawer extends MenuDrawer {
   constructor() {
     super();
+
+    this.onResize = () => {
+      document.documentElement.style.setProperty('--header-bottom-position', `${parseInt(this.header.getBoundingClientRect().bottom - this.borderOffset)}px`);
+      document.documentElement.style.setProperty('--viewport-height', `${window.innerHeight}px`);
+    };
```

Comment:
I can see that what we did yesterday didn't work (`onResize() {`) because `this` wasn't bound. However I think it's cleaner to use a class method with an arrow function, which is also valid and will automatically bind `this`:
```js
onResize = () => {
  ...
}
```

Reply:
Yeah, and I tried even .bind(this) and it didn't work. But I didn't try to use arrow function instead of a normal one. Now this works and look better I agree. Thanks.